ipynb/AG_stool_temporal dynamics.ipynb

```diff
@@ -0,0 +1,664 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import skbio\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "dm = skbio.DistanceMatrix.read('ag-cleaned-multiple-timepoints_even1250_unweighted.txt')\n",
+    "w_dm = skbio.DistanceMatrix.read('ag-cleaned-multiple-timepoints_even1250_weighted.txt')\n",
+    "deltas = pd.read_csv('ag-cleaned-multiple-timepoints-deltas.txt', sep='\\t', dtype=str)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "absdeltas = deltas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "deltas['DeltaInDays'] = pd.to_numeric(deltas['DeltaInDays'])\n",
+    "absdeltas['DeltaInDays'] = deltas['DeltaInDays'].abs()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "565\n",
+      "1553\n",
+      "1553\n"
+     ]
+    }
+   ],
+   "source": [
+    "unique_ids = []\n",
```

Comment:
You can replace the following three for-loops for a call to the `unique` method i.e.

```python
unique_ids = deltas.HOST_SUBJECT_ID.unique()
```